src/components/ui/bottomNavigation/bottomNavigationTab.component.tsx

```diff
@@ -23,19 +25,23 @@ import {
   TextElement,
 } from '../text/text.component';
 import { IconElement } from '../icon/icon.component';
-import { isValidString } from '../support/services';
+import {
+  isValidString,
+  WebEventResponder,
+  WebEventResponderCallbacks,
+  WebEventResponderInstance,
+} from '../support/services';
 
 type IconProp = (style: ImageStyle) => IconElement;
 
-interface ComponentProps {
+export interface BottomNavigationTabProps extends StyledComponentProps, TouchableOpacityProps {
```

Comment:
I believe that the previous version is slightly easier to read. But it's OK to move on with it.

Reply:
sorry, just tried to unify 🤷‍♂